src/app/utils/Links.js

```diff
@@ -35,8 +35,42 @@ export default {
     ipfsPrefix: /(https?:\/\/.*)?\/ipfs/i,
 }
 
+//todo: possible this should go somewhere else.
```

Comment:
1. please write TODO, not todo
2. please resolve these TODOs. where else would this code go?

Reply:
Most likely in a file called utils/Url.js that's currently in the `notifications-ui` branch. Some recent conversations with @bnchdrff suggest that file may be changing or going away.

The TODO is not currently resolvable, but I will capitalize the word for you.

Reply:
if we haven't formally decided to deprecate the file, that's where the code should go.